build-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17.5-buster
+FROM golang:1.17.8-buster
```

Comment:
Change to build image requires rebuild. I've done that for you: `quay.io/cortexproject/build-image:update-go-1-17-8-6aed4de76`.